### PR TITLE
Fix macOS fullscreen incorrect mouse position

### PR DESCRIPTION
**Pull request by danielkariv**
In relation to #40061. 

This PR is fixing the incorrect mouse position in fullscreen mode which result UI click registered below expect point.
From my testing, it seems like the issue is that when the window getting into or out fullscreen mode, it wouldn't call the resize event to get correct sizing and scale.
I also notice that when i fullscreen and changing resolution of the display, it will call resize event, and the UI will work as it should. but when getting out from fullscreen mode, it will get back to the same issue, now in window mode.

So my PR is adding a call to resize event when changing to/from fullscreen mode.

### CODE REVIEW

File: platform/osx/display_server_osx.mm [hunk 1]
```diff
@@ -190,6 +190,8 @@ - (void)windowDidEnterFullScreen:(NSNotification *)notification {
 
 	[wd.window_object setContentMinSize:NSMakeSize(0, 0)];
 	[wd.window_object setContentMaxSize:NSMakeSize(FLT_MAX, FLT_MAX)];
+	//Force window resize event
```
Comment (akien-mga) [inline platform/osx/display_server_osx.mm:193]: Nitpick, but we usually style our comments like this:

```suggestion
	// Force window resize event.
```

**Comment by danielkariv** [comment]
Change the comment to fit with the style you requested.

**Review by bruvzg — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thanks for the fix! Can you squash your comment fixup commit into the first one?

Then this should be good to go.

**Comment by danielkariv** [comment]
I am sorry but I am new to using git and I think can't do that anymore. I thought the PR got approved so i had remove the fork/branch from account and now I can't figure out how to restore/get the pr branch back. If it isn't possible to do from your end, I could try to redo it with a new PR.

**Comment by mhilbrunner** [comment]
> I am sorry but I am new to using git and I think can't do that anymore. I thought the PR got approved so i had remove the fork/branch from account and now I can't figure out how to restore/get the pr branch back. If it isn't possible to do from your end, I could try to redo it with a new PR.

So, before doing anything, make sure you make a backup of your changes. :)

1. Make sure you're in the right branch (the one for this PR)
2. Git rebase, and select "squash" for the second commit (the one with the comments)
3. Finish rebasing, commit, force push

If you need a more detailed tutorial, searching for the above should help.
But yes, if its easier for you you can also open a new PR and reference this one and then close this one - but really, thats not necessary if you follow the above. And if you need any help, let us know on chat.godotengine.org

**Pull request closed without merging**